br-webui/views/camera.liquid

```diff
@@ -478,9 +542,146 @@
+			
+			streamTypeSelect.appendChild(option)
+		}
+		
+		// Menu under streamTypeSelect with all options for each stream type
+		var streamTypeMenu = document.getElementById('stream-type-menu')
+		//UDP
+		{
+			// Create a parent item to enable visibity changes of the sub items
+			let udpMenu = document.createElement("div")
+			udpMenu.id = 'udp-menu'
+
+			let title = document.createElement("h4")
+			title.innerHTML = "Clients: "
+
+			let input = document.createElement("input")
+			input.id = 'udp-clients'
+			input.value = '192.168.2.2:5600'
+			
+			let info = document.createElement("div")
+			info.class = 'alert alert-info'
+			info.role = 'alert'
+			info.innerHTML = 'Accepts multiple endpoints, E.g: 192.168.2.2:5600,192.168.2.3:5600,...'
+			
+			title.appendChild(input)
+			udpMenu.appendChild(title)
+			udpMenu.appendChild(info)
+			streamTypeMenu.appendChild(udpMenu)
+		}
+
+		// RTSP
+		{
+			// No menu for now
+		}
+		
+		// Update visibility
+		streamTypeChanged()
+	}
+	
+	/// Check if the video stream type changed
+	function streamTypeChanged() {
+		// Get component and selected option
+		const streamTypeSelect = document.getElementById("stream-type-select")
+		const option = streamTypeSelect.options[streamTypeSelect.selectedIndex]
+		
+		// Use the selected option to make the `option-menu` visible and others hidden
+		for (const [streamType, _] of Object.entries(PIPELINES)) {
+			const menuName = streamType + '-menu'
+			var item = document.getElementById(menuName)
+			if (item == null) {
+				console.log(`Component: ${menuName} does not exist or is not ready yet.`)
+				break
+			}
+			const visible = streamType == option.innerHTML
+			item.style.visibility = visible ? 'visible' : 'hidden'
+			item.style.display = visible ? 'block' : 'none'
+		}
+	}
+
+	// Set default gstreamer configuration
+	function setDefaultGstreamerConfiguration() {
+		const streamType = "udp"
+		const frameSize = [1920, 1080].join('x')
+		const frameRate = 30
+
+		changeStreamType(streamType)
+
+		console.log(`Updating selected frame size to: ${frameSize}`)
+		$('#frame-size-select').val(frameSize).change()
+
+		console.log(`Updating selected framerate to: ${frameRate}`)
+		$('#framerate-select').val(frameRate).change()
+
+		applyGstreamer()
+	}
+
+	// Apply Gstreamer pipeline
+	function applyGstreamer() {
+		// Get selected stream option
+		const streamTypeSelect = document.getElementById("stream-type-select")
+		const option = streamTypeSelect.options[streamTypeSelect.selectedIndex]
+		const stream_type = option.innerHTML
+		
+		/// Crete necessary variables for the pipeline
+		// Generic variables
+		const camera = getCameraSelection()
+		const [width = 1920, height = 1080] = getFrameSizeSelection().split('x')
+		const framerate = getFpsSelection()
+		
+		const clients = (function() {
+			switch (stream_type) {
+				case 'udp':
+					return document.getElementById("udp-clients").value
+					break
+				case 'rtsp':
+					return '0.0.0.0:8554/video1'
```

Comment:
```suggestion
					return window.location.hostname + ':8554/video1'
```

This doesn't work at all for me with 0.0.0.0

Reply:
Nice catch, but I'll fix that in camera-manager side since this field is optional

Reply:
The service will catch any string with {service_ip} and update the value for a valid ip for qgc.
`return '{service_ip}:8554/video1'`